### PR TITLE
Remove .only from test

### PR DESCRIPTION
**Pull request by gabegorelick**
This was causing only one test to be run.

### CODE REVIEW

**Comment by gabegorelick** [comment]
You may also want to add https://github.com/lo1tuma/eslint-plugin-mocha/blob/47f624abfac84b963007423cf5a35ca46ad4d981/docs/rules/no-exclusive-tests.md to prevent this from happening in the future.

File: test/test_repeat.js [hunk 1]
```diff
@@ -644,7 +644,7 @@ describe('repeat', function() {
       );
   });
 
-  it.only('should emit a waiting event when adding a repeatable job to the waiting list', function(done) {
+  it('should emit a waiting event when adding a repeatable job to the waiting list', function(done) {
```
Comment (gabegorelick) [inline test/test_repeat.js]: Perhaps this was supposed to be a `.skip` since this test is failing?

**Comment by gabegorelick** [comment]
Build is failing because this test doesn't pass. Perhaps it should be skipped?

File: test/test_repeat.js (hunk 1 above)
Comment (manast) [reply test/test_repeat.js]: must have been committed while trying to fix the reason for the test to fail

**Comment by manast** [comment]
I am gonna change our merge procedure, having a "develop" branch and only merge to master when we do a new release, so that master is kept green all the time.

**Pull request merged**